data_structures/src/builders.rs

```diff
@@ -75,12 +78,26 @@ impl Message {
             last_epoch,
             genesis: GENESIS,
             nonce: random_nonce(),
-        })
+        }))
     }
 
     /// Function to build Verack messages
     pub fn build_verack() -> Message {
-        Message::build_message(Command::Verack)
+        Message::build_message(Command::Verack(Verack))
+    }
+
+    /// Function to build Inv messages
+    pub fn build_inv(mut inv_elems: Vec<InvElem>) -> Message {
+        // Get max number of inv elements that can be sent
+        let num_elems = inv_elems.len() as u16;
```

Comment:
This is not safu.

Reply:
:see_no_evil: shame on me, I'll change it now